test/e2e/dns_autoscaling.go

```diff
@@ -23,6 +23,7 @@ import (
 	"strings"
 	"time"
 
+	"k8s.io/kubernetes/pkg/api"
```

Comment:
Turns out in #36673 @caesarxuchao was changing the entire e2e tests to use versioned api. Will remove this and figure out another way to retrieve `api.NamespaceSystem`.